docs/en/functionalities.rst

```diff
@@ -0,0 +1,413 @@
+
+- **Remote Presentation**: The User presents the PID and/or a set of Attributes contained in one or more Electronic Attestations through the Wallet Instance, to a Relying Party configured for online verification, for instance, to Authenticate and access the services offered. 
+
+
+Proximity Presentation 
+=======================
+
+Proximity presentation allows the User to present the PID and/or a set of Attributes contained in one or more Electronic Attestations via their Wallet Instance, using one of two methods: 
+
+- **Supervised mode**: The User presents the PID and/or a set of Attributes contained in one or more Electronic Attestations through the Wallet Instance to a Verifier equipped with a dedicated verification app or system (e.g., law enforcement officer, pharmacist); 
+
+- **Unsupervised mode**: The User presents the PID and/or a set of Attributes contained in one or more Electronic Attestations through the Wallet Instance to a designated device (e.g., turnstile, ATM). 
+
+Below are the functional requirements supporting the User Experience for both methods.
+
+**Supervised Mode** 
+
+- The User MUST access their Wallet Instance using the unlock method previously set; 
+- The User MUST navigate to the feature dedicated to QR Code generation; 
+- The User MUST present the generated QR Code to the Verifier acting on behalf of the Relying Party, who scans it using the designated verification app or system; 
+- The User MUST review the requested PID data and/or Attributes, the name of the requesting Service Provider, and any related policy. The User MUST decide whether to share any non-mandatory PID data and/or Attributes (Selective Disclosure). The User MUST provide consent to proceed or MAY cancel the operation; 
+- The User MUST authorize the operation using the unlock method previously set; 
+- The User MUST receive confirmation of the successful presentation. 
+
+If an error occurs, the User MUST receive clear and consistent messages informing them of the issue and guiding them toward resolution. For further details, please refer to the `Error Management`_ section. 
+
+For further details and resources, please refer to www.wallet.gov.it.  
+
+**Unsupervised Mode** 
+
+- The User MUST access their Wallet Instance using the unlock method previously set; 
+- The User MUST navigate to the feature dedicated to QR Code generation; 
+- The User MUST present the generated QR Code to the designated device (e.g., a turnstile) of the Relying Party for scanning; 
+- The User MUST review the requested PID data and/or Attributes, the name of the requesting Relying Party, and any related policy. The User MUST decide whether to share any non-mandatory PID data and/or Attributes (Selective Disclosure). The User MUST provide consent to proceed or MAY cancel the operation; 
+- The User MUST authorize the operation using the unlock method previously set; 
+- The User MUST receive confirmation of the successful presentation. 
```

Comment:
if we use `;` at the end of line -> `the User` and not `The User`

otherwise we should put a `.`